### PR TITLE
Make CLI output same as in other SDKs

### PR DESCRIPTION
**Pull request by ghorbanzade**

### CODE REVIEW

**Comment by ghorbanzade** [comment]
Just some minor formatting changes to the CLI output so that the output is visually identical with other SDKs.

File: touca/src/main/java/io/touca/runner/Runner.java [hunk 1]
```diff
@@ -143,10 +160,11 @@ public Runner parse(final String[] mainArgs) {
...
 
-      final BiFunction<String, Boolean, Boolean> parseBoolean = (String a, Boolean b) -> {
-        return (cmd.hasOption(a) && cmd.getOptionValue(a) == null)
-            || Boolean.parseBoolean(cmd.getOptionValue(a, b.toString()));
-      };
+      final BiFunction<String, Boolean, Boolean> parseBoolean =
+          (String a, Boolean b) -> {
+            return (cmd.hasOption(a) && cmd.getOptionValue(a) == null)
+                || Boolean.parseBoolean(cmd.getOptionValue(a, b.toString()));
+          };
```
Comment (ghorbanzade) [inline touca/src/main/java/io/touca/runner/Runner.java:167]: We need to figure a way to enforce the same code formatting in both of our IDEs. It appears our IDEs are fighting with each other, at the moment.

I'll add a ticket for us to find a well-known code formatter that we can integrate to resolve this issue.

File: touca/src/main/java/io/touca/runner/Color.java [hunk 2]
```diff
@@ -1,28 +1,31 @@
 package io.touca.runner;
 
 public enum Color {
```
[the remark this answers is not in the document]
Comment (Mdkhaki) [reply touca/src/main/java/io/touca/runner/Color.java:3]: I didn't know why people do that. Good.

File: touca/src/main/java/io/touca/runner/Runner.java (hunk 1 above)
Comment (Mdkhaki) [reply touca/src/main/java/io/touca/runner/Runner.java:167]: That's very irritating.

File: touca/src/main/java/io/touca/runner/Runner.java [hunk 3]
```diff
@@ -60,49 +61,65 @@ public Long count(final String key) {
...
+              (sum, item) -> Math.max(sum, item));
+      this.testcaseCount = options.testcases.length;
     }
 
-    public void printHeader() {
-      System.out.printf("%nTouca Test Framework%nSuite: %s/v%s%n%n",
-          options.suite, options.version);
+    private void print(final String fmt, Object... args) {
+      System.out.printf(fmt, args);
     }
```
[the remark this answers is not in the document]
Comment (Mdkhaki) [reply touca/src/main/java/io/touca/runner/Runner.java:76]: This is a smart move.

**Pull request merged**